lib/active_merchant/billing/gateways/creditcall.rb

```diff
@@ -0,0 +1,204 @@
+require 'nokogiri'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class CreditcallGateway < Gateway
+      self.test_url = 'https://test.cardeasexml.com/generic.cex'
+      self.live_url = 'https://live.cardeasexml.com/'
+
+      self.supported_countries = ['US']
+      self.default_currency = 'USD'
+      self.supported_cardtypes = [:visa, :master, :american_express, :discover]
+
+      self.homepage_url = 'https://www.creditcall.com'
+      self.display_name = 'Creditcall'
+
+      def initialize(options={})
+        requires!(options, :terminal_id, :transaction_key)
+        super
+      end
+
+      def purchase(money, payment_method, options={})
+        MultiResponse.run(:use_first_response) do |r|
```

Comment:
I don't think you want to use `: use_first_response` here.  For `verify` that makes sense since we don't care about the `void`.